R/google_drive.R

```diff
@@ -356,21 +356,7 @@ gd_confirm_posted <- function(
   remote_info <- remote_path %>% slice(n()) %>% pull(drive_resource) %>% .[[1]]
   
   # we could prepare a timestamp from modifiedTime...but checksum is even better
-  indicate(ind_file, md5_checksum=remote_info$md5Checksum)
+  sc_indicate(ind_file, md5_checksum=remote_info$md5Checksum)
```

Comment:
current behavior is to use the key `hash` instead of `md5_checksum` in sc indicator files - i think all that's needed is to change the argument name